microsoft-edge/privacy-whitepaper.md

```diff
@@ -0,0 +1,427 @@
+---
+description: Microsoft Edge Privacy Whitepaper
+title: Microsoft Edge Privacy Whitepaper
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 01/15/2020
+ms.topic: article
+ms.prod: microsoft-edge
+ms.technology: browser
+keywords: microsoft edge, privacy, whitepaper, trust
+---
+
+# Microsoft Edge Privacy Whitepaper
+
+[Address bar and suggestions](#address-bar-and-suggestions) | [Autofill](#autofill) | [Cast](#cast) | [Collections](#collections) | [Crashes](#crashes) | [Diagnostic data about how you use the browser](#diagnostic-data-about-how-you-use-the-browser) | [Digital Rights Management and Media Licenses](#digital-rights-management-and-media-licenses) | [Do Not Track](#do-not-track) | [Downloads](#downloads) | [Extensions and Microsoft Edge Addons](#extensions-and-microsoft-edge-addons) | [Family safety](#family-safety) | [Feedback](#feedback) | [Geolocation](#geolocation) | [Install and update](#install-and-update) | [Internet Explorer mode](#internet-explorer-mode) | [Intrusive ads](#intrusive-ads) | [Jump list](#jump-list) | [Network time](#network-time) | [New tab page](#new-tab-page) | [On startup](#on-startup) | [Payments](#payments) | [Personalizing advertising, search, news and other Microsoft services](#personalizing-advertising-search-news-and-other-microsoft-services) | [Print](#print) | [Profiles](#profiles) | [Read aloud](#read-aloud) | [Resolve navigation errors](#resolve-navigation-errors) | [Sign in and Identity](#sign-in-and-identity) | [SmartScreen](#smartscreen) | [Spellcheck](#spellcheck) | [Suggest similar sites](#suggest-similar-sites) | [Sync](#sync) | [Testing new functionality](#testing-new-functionality) | [Tracking prevention](#tracking-prevention) | [Translate](#translate) | [Web apps and Pinned sites](#web-apps-and-pinned-sites) | [WebView](#webview) | [Websites you visit for product and service improvement](#websites-you-visit-for-product-and-service-improvement) | [Windows Defender Application Guard](#windows-defender-application-guard) | [Windows Information Protection](#windows-information-protection)
+
+Our browser privacy promise is to provide you with the protection, transparency, control and respect you deserve.  To uphold our commitment to give you transparency into our product, we’ve provided a privacy whitepaper that explains how Microsoft Edge features and services work and how they may affect your privacy.  Our goal is to give you a full understanding into how your data is used, how to control the different features, and how to manage your collected data, so you have the info you need to make the right privacy decisions for you.  
```

Comment:
Replace all smart punctuation with ascii equivalent to prevent build issues. 
Replace `’` with `'`.
Replace `“` with `"`.
Replace `”` with `"`. And so on.